content/blog/2024-05-20-summer-software-open-science-school.md

```diff
@@ -0,0 +1,59 @@
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering. 
+To that end, we strongly encourage attendees to consider a particular software project to apply learning principles to during the school. 
+Throughout the sessions, learners will collaborate with other school participants on their software projects and apply software engineering and open science best practices to make their work visible, citable, and accessible.    
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+All disciplines are welcome at this school; including--but not limited to--practitioners in the sciences, engineering, humanities, social sciences and economics.  
+If you use or develop software in the course of doing research, you will find applicable skills in this workshop for your work. 
+
+## Target Audience
+
+Ideal candidates for this workshop are science practitioners who use or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
+These practitioners want to make their research open, accessible, and reproducible by implementing open science best practices in addition to building and contributing to research software. 
+
+To get the most benefit from this workshop, we expect students to be familiar with the Unix shell, Python, and git, at the level taught at a [Software Carpentry](https://software-carpentry.org/) Workshop. 
+
+## Format and topics
+
+This five-day workshop will enable learners to hone their skills in developing sustainable research software, practicing open science in their workflow, and contributing to their communities of practice. Topcs covered will include:  
+
+*   The Ethos of Open Science
+*   Open Tools and Resources
+*   Open Data 
+*   Open Results 
+*   Software design and modularity
+*   Collaborative software development via Git+GitHub
+*   Software testing in Python
+*   Code review
+*   Packaging and distributing Python software
+*   Documentation
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant is strongly encouraged to bring a project to work on throughout the course for applying these concepts.
```

Comment:
same comment as above - I recommend making bringing a project or things to work on mandatory (and part of the application)